packages/elements/src/pagination/index.ts

```diff
@@ -666,12 +636,18 @@ export class Pagination extends BasicElement {
         this.hasLastPage() && this.updateInputValue(this.internalMax);
         break;
       default:
-        return;
     }
 
-    if (event.key !== 'Tab') {
-      event.preventDefault();
+    /**
+     * Allow input keys (navigation, submit, number).
+     * Do not prevent default for `Tab` key, it should focus on next tabindex as the same native behaviour.
+     */
+    const allowKeys = ['ArrowLeft', 'ArrowRight', 'Backspace', 'Tab'];
```

Comment:
I can see at least two problems with this:
1) It does not support IE11, which has 'Left' and 'Right'
2) You can still enter invalid value, e.g. by Copy/Paste

I would implement beforeinput event to strip down all non-numeric characters. But, for IE11 (which does not support beforeinput) you can either ignore (as technically entering letters instead of numbers is a user error) or on input check the value and strip off all non numeric characters.

Reply:
1. Fixed
2. We can use `onPaste` event to solve this issue it very simple.